Applications/CMakeLists.txt

```diff
@@ -99,6 +99,7 @@ if (MODULE_Registration)
   endmacro()
 
   add_registration_command(register LibIO OPTIONAL LibLBFGS LibPointSet LibDeformable)
+  add_registration_command(create-sdm LibIO)
```

Comment:
See my other comment. I don't think so. a SDM has a lot to do with your statistical FFD. It **is** a model for such FFD, or, one particular instance of it.